app/assets/stylesheets/application.scss

```diff
@@ -18,10 +18,42 @@
 
 body {
   background-color: #e0e0e0;
+  background-image: url("/assets/woody.jpg");
+  // background-repeat: no-repeat;
+  position: relative;
```

Comment:
Properties should be ordered background-color, background-image, margin-bottom, margin-top, position